### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by valentinastron** [PluginsOCSInventory-NG/officepack#19]
Hi, 
I tried to install this officepack plugin, but I get an error message, it's contains the following:

ERROR: officepack-master is an invalid plugin, check your sources.
Installation aborted !
![ocs](https://cloud.githubusercontent.com/assets/25320898/22244867/80afc92c-e22d-11e6-93f3-d1bc26078fff.PNG)
This plugin updated 15 days ago, so I think this is the latest version. 
Can somebody help me how can I install this plugin? 
Regards.

**Comment by valentinastron** [comment]
I found the bug fix in another issue, so now I installed the plugin but I get another warning
The communication server encountered the following error on install :  
Click here to solve your problem.
I checked the link but this article didn't help me... 
Copied the .vbs file to the C:\Program Files (x86)\OCS Inventory Agent\Plugins directory but I have not received office key informations.

**Comment by gillesdubois** [comment]
@valentinastron which error code did u encountered ?

**Comment by valentinastron** [comment]
none of them.  
![kepkivagas](https://cloud.githubusercontent.com/assets/25320898/22247702/79b28ce6-e23b-11e6-8418-2ca7c136de30.PNG)

**Comment by valentinastron** [comment]
I ran the agent and the vbs script generated an error log in C:\ and it contains the following: 
Input error: Can't find (C:\Program Files\Microsoft Office\Office16\OSPP.VBS).
I have Office15 directory there and the OSPP.VBS script is there.

**Comment by valentinastron** [comment]
Update:
Installed a new server ( the other was an update) and now I get the Err_04. I did what the article said but I get the same error. What should I do ?

**Comment by valentinastron** [comment]
Update2:
Problem with the Communication server:
I changed Require all granted line to Allow from All in apache conf after that the error message disappeared but the /etc/ocsinventory/perl/Apache/Ocsinventory/Plugins dir deleted. When I recreated this and installed the plugin again everything was fine, no error message.
I moved the msofficekex.vbd to the agent plugin dir and run the agent. The inventory data arrived but the office key didn't.

No error message, no log.
Any idea?
